google_reauth/reauth.py

```diff
@@ -0,0 +1,276 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A module that provides functions for handling rapt authentication."""
+
+import base64
+import getpass
+import json
+import logging
+import sys
+
+from six.moves import urllib
+
+from pyu2f import errors as u2ferrors
```

Comment:
why not just `import pyu2f.errors` and use `pyu2f.errors` throughout?

Actually, apply this to all pyu2f imports.